data/governments.txt

```diff
@@ -970,11 +970,34 @@ government "Indigenous Lifeform (Astral)"
 		destroy 0
 		atrocity 0
 
+government "Indigenous Lifeform (Subsidurial)"
+	# This indigenous gov is specifically for the Subsidurial, so that nothing in the game tries to hurt the poor thing.
+	"player reputation" 1
+	"bribe" 0
+	"fine" 0
+	
+	"default attitude" 0.01
```

Comment:
Get rid of this. The government they currently use has an attitude of zero towards everything except the "Ember Waste" government.